docs/roadmap/cosmos-hub-roadmap-2.0.md

```diff
@@ -65,20 +66,21 @@ The upgrades aim to add features such as liquidity, economic security, usability
   - The Cosmos solution to shared security that uses IBC Cross Chain Validation (CCV) to relay validator set composition from a Provider Chain (Cosmos Hub) to a Consumer Chain. This validator set is in charge of producing blocks on both networks using separate nodes. Misbehavior on the Consumer Chain results in slashing Provider Chain staking tokens (ATOM).
   - Allows independent modules like Gravity DEX or Bridge to live on separate chains with their own development cycles.
 - Chain Name Service
-  -  Chain-ID registry
-    - Node registry
-    - IBC Path Resolution
-  -  Asset registry
-  -  Account registry
-  -  Bech32 registry
+  - Chain-ID registry
+  - Node registry
+  - IBC Path Resolution
+  - Asset registry
+  - Account registry
+  - Bech32 registry
 
 
 ## Lambda Upgrade (expected Q3 2022)
```

Comment:
IBC queries @jackzampolin ?

Reply:
need to link up w IBC team to confirm their roadmap before speaking on their behalf. Or is the query work confirmed to come from an external group??